src/ffjord.jl

```diff
@@ -1,11 +1,35 @@
 abstract type CNFLayer <: Function end
 Flux.trainable(m::CNFLayer) = (m.p,)
 
+struct DeterministicCNFLayer{M,P,RE,Distribution,T,A,K} <: CNFLayer
```

Comment:
the other layers don't have this Layer in the name. For consistency it should be dropped.

Reply:
Ok.